### DISCUSSION TITLE
Get page route in getStaticProps for non-dynamic pages

### DISCUSSION THREAD

**Discussion by barrard** [vercel/next.js#33157] [Help]
Hey all,

I'm looing for a way to get the page route for my static, non-dynamic pages.  
For example, in /pages/about/index.js, inside my getStaticProps functions, I'd like the functions to "know" it's for the /about route.  Is this possible?  Thank you!

**Comment by sergeyzenchenko** [comment]
I need the same, can't found any solution yet, but inside Next server in renderToHtml function there is a pathname that can be passed into getStaticProps. Not sure why it's not done this way
